docs/optimal_control.rst

```diff
@@ -66,7 +66,7 @@ We can look at the parameter values this opt_map specified with
 
 .. code-block:: python
 
-    from c3.optimizers.c1 import C1
+    from c3.optimizers.optimalcontrol import C1
```

Comment:
Is this a typo? `C1 -> OptimalControl`

Reply:
Ah, yes. The markdown wasn't refactored. I'll do it.